dropwizard-views-mustache/src/main/java/io/dropwizard/views/mustache/MustacheViewRenderer.java

```diff
@@ -26,6 +28,7 @@
 public class MustacheViewRenderer implements ViewRenderer {
     private final LoadingCache<Class<? extends View>, MustacheFactory> factories;
     private boolean useCache = true;
+    private File fileRoot;
```

Comment:
I think it would be better to declare this field as `Optional` and initialize with `Optional.empty`. Doing so, we explicitly state that field's value can be absent.